scripts/README.md

```diff
@@ -1,5 +1,5 @@
 tsplot
```

Comment:
Both are valid markdown, but I'll change to '#' for consistency with the other documents.